src/modules/transaction/store/actions.js

```diff
@@ -145,15 +145,15 @@ export const transactionBroadcasted = (transaction, moduleCommandSchemas) =>
     const activeToken = token.active;
     const serviceUrl = network.networks[activeToken].serviceUrl;
     let broadcastResult;
-    // @todo dry run before broadcast
-    const dryRunResult =  await dryRun({ transaction, serviceUrl, network });
+    const dryRunResult = await dryRun({ transaction, serviceUrl, network });
 
-    if (dryRunResult.data?.success === true) {
+    // If the results is 1 then its successful
```

Comment:
I don't think we need this already 

`if (dryRunResult.data?.result === 1) {` the code already says it all